examples/with-eip-1193-provider/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@turnkey/example-with-eip-1193-provider",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@radix-ui/react-dialog": "^1.0.5",
+    "@radix-ui/react-dropdown-menu": "^2.0.6",
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-label": "^2.0.2",
+    "@radix-ui/react-select": "^2.0.0",
+    "@radix-ui/react-slot": "^1.0.2",
+    "@radix-ui/react-toast": "^1.1.5",
+    "@turnkey/api-key-stamper": "workspace:^",
+    "@turnkey/eip-1193-provider": "workspace:*",
+    "@turnkey/http": "workspace:^",
+    "@turnkey/webauthn-stamper": "workspace:^",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.0",
+    "lucide-react": "^0.363.0",
+    "next": "14.1.4",
+    "next-themes": "^0.3.0",
+    "react": "^18",
+    "react-dom": "^18",
+    "sonner": "^1.4.41",
+    "tailwind-merge": "^2.2.2",
+    "tailwindcss-animate": "^1.0.7",
+    "viem": "^2.9.5"
```

Comment:
oh wow they're already on 2.9.x! total side note: we should bump our internal viem dependency soon